src/applications/ezr/components/IntroductionPage/SaveInProgressInfo.jsx

```diff
@@ -30,6 +30,7 @@ const SaveInProgressInfo = ({ formConfig, pageList }) => {
   // set the props to use for the SaveInProgressIntro components
   const sipProps = {
     startText: content['sip-start-form-text'],
+    startPage: '/veteran-information/personal-information',
```

Comment:
I know this is temporary, but I think it might be cleaner to pull this route from the form config. So we could add `chapters` to the destructure above... then this value just becomes `chapters.veranInformation.pages.profileInformation`.

Reply:
Great suggestion Matt thanks so much for this tidbit!